### PR TITLE
[Group] Overhaul Group page layout

### PR DESCRIPTION
**Pull request by rkdavidson**
# Related Issue
- [**🏕 Make group page layout responsive**](https://3.basecamp.com/3926363/buckets/17566548/todos/3185252598)

# Changes or Updates
- Duplicates, then overhauls the group layout _on the sandbox version_ of the site
- Adds members list with a "See all" modal

# Tradeoffs/Caveats
This is still somewhat of a WIP layout.
There are some fairly obvious refinements that could be made to responsive re-stacking of elements, vertical alignments, and the polish/finessing on the new Members List.
This PR/scope is intended to improve what's there now, and add more information missing from the web currently for groups.

# Test Instructions
- Verify individual groups pages load and generally function properly both _with_ and _without_ sandbox mode enabled (`REACT_APP_ENABLE_SANDBOX=true` / `REACT_APP_ENABLE_SANDBOX=`)
- Verify responsive sizes are generally acceptable
- Verify Add to Calendar event description/config is generally acceptable (note: existing issue with Apple calendar on Chrome for some users)

# Screenshots

|Description|Screenshot|
|---|---|
|Sandbox mode disabled||
|New Layout - **Small**||
|New Layout - **Medium**||
|New Layout - **Large**||
|Members list and See All||

# Tests
- [ ] needed snapshots updated
- [ ] mobile tested
- [ ] latest update from master

### CODE REVIEW

**Review by JSGraef — COMMENTED** [review]
Can't seem to get the new layout to show. I have `REACT_APP_ENABLE_SANDBOX=true` but I only see the original layout.

**Review by JSGraef — APPROVED** [review]
Got it. Yeah this works well - happy to merge this in and address things as +1s.

File: src/groups/NewGroupContentItemConnected/GroupMembers.js [hunk 1]
```diff
@@ -1,33 +1,49 @@
...
-            />
-            <div style={{ fontSize: '0.75rem' }}>
-              {get(member, 'nickName') || get(member, 'firstName')}
-            </div>
-          </div>
-        ))}
-      </div>
-    </div>
+    <MembersList showAll={showAll}>
+      {members.slice(0, displayCount).map((member, index) => (
```
[the remark this answers is not in the document]
Comment (rkdavidson) [reply src/groups/NewGroupContentItemConnected/GroupMembers.js:40]: Yes, great call out (as with the other comments here. Given this is sandbox'ed, so only accessible on beta site and locally, this isn't an immediate concern but definitely something we should do prior to full release.

Captured here:

[**🏕 Use the pagination API for Group Members**](https://3.basecamp.com/3926363/buckets/17566548/todos/3197044617)

**Review by JSGraef — APPROVED** [review]
👍

**Pull request merged**